docs/source/nse_table.ipynb

```diff
@@ -0,0 +1,250 @@
+      "   9.0000000000    9.6000000000    0.4300000000   11.0638296395    8.1412201568   1.0225013e-08  -9.8142004e-34  -5.2463937e-15   1.1664591e+10 \n",
+      "   9.0000000000   10.0000000000    0.5000000000    4.3121615772    7.1515898299     -0.10742116  -7.8751776e-19   2.9757777e-05   4.7264232e+17 \n",
+      "   9.0000000000   10.0000000000    0.4650000000    5.1247240232    7.3995844381    -0.019430675   -5.218123e-20   3.3567416e-06   8.7381876e+16 \n",
+      "   9.0000000000   10.0000000000    0.4300000000    4.4162729378    7.0777579412   -0.0059568824   1.4777727e-20   5.1854108e-07   3.2607638e+16 \n",
+      "   9.0000000000   10.4000000000    0.5000000000    1.0000214391    0.0002025494      -17.715252  1.5106478e-112   3.7266649e-96   2.7735801e+20 \n",
+      "   9.0000000000   10.4000000000    0.4650000000    1.0000212298    0.0002005604      -15.005795 -5.5593438e-112   3.0826278e-96   2.5962747e+20 \n",
+      "   9.0000000000   10.4000000000    0.4300000000    1.0000206076    0.0001946699      -12.327315 -6.3674509e-112   1.9400005e-96   2.4324142e+20 \n",
+      "  10.0000000000    9.6000000000    0.5000000000   55.8692197296    8.6423237057      -5.2276094              -0       0.2720124   2.9376254e+19 \n",
+      "  10.0000000000    9.6000000000    0.4650000000   55.9152992729    8.7874226105  -0.00092940692   -2.648266e-16   8.5387004e-05   4.3026908e+15 \n",
+      "  10.0000000000    9.6000000000    0.4300000000   11.0638297827    8.1412201659   5.0971987e-15  -9.3107493e-42  -6.8646484e-21       5801.4456 \n",
+      "  10.0000000000   10.0000000000    0.5000000000   13.9370868136    8.2743459363      -5.5785841   1.0782619e-14      0.10961074   3.9381535e+19 \n",
+      "  10.0000000000   10.0000000000    0.4650000000   28.2438967467    8.6492551831    -0.075078082   2.0321472e-16   0.00050736054   5.2201774e+17 \n",
+      "  10.0000000000   10.0000000000    0.4300000000   10.2419420841    8.0844779277   -0.0025107223  -1.1694167e-19   1.1073417e-06   1.7236419e+16 \n",
+      "  10.0000000000   10.4000000000    0.5000000000    1.0197762456    0.1829100336       -196.6445   -4.638832e-57     3.90578e-42   2.3555844e+21 \n",
+      "  10.0000000000   10.4000000000    0.4650000000    1.0195858994    0.1811833443      -167.54221   3.2294093e-57   3.0442073e-42   1.9968628e+21 \n",
+      "  10.0000000000   10.4000000000    0.4300000000    1.0190251146    0.1760925439      -140.99607  -1.1373217e-57   1.8183284e-42   1.6720803e+21 \n"
+     ]
+    }
+   ],
+   "source": [
+    "%cat nse.tbl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00b27893-a7ed-43e2-b608-b7f381b0fa6d",
+   "metadata": {},
+   "source": [
+    "There are other options to the NSE table writing, including storing the mass fractions or a reduced composition constructed by binning the nuclei\n",
+    "into the closest nuclei in a smaller set."
```

Comment:
Is there a way to get it to output the mass fractions without binning the composition?

Reply:
not currently, since I did not see that as a use case, but we can add that functionality in another PR.

Reply:
Alright